.nightfalldlp/config.json

```diff
@@ -2,5 +2,6 @@
     "detectors": {
         "CREDIT_CARD_NUMBER": "POSSIBLE",
         "PHONE_NUMBER": "LIKELY"
-    }
+    },
+    "max_number_concurrent_routines": 5
```

Comment:
why did we choose 5?

Reply:
I was testing with test data which required 15 API requests so I used 5 to ensure that the number of go routines was working correctly. 

I can update this if we want to change it.